docs/en/compute/AWS-EMR.md

```diff
@@ -40,7 +40,9 @@ To begin with, [download an Alluxio release](https://www.alluxio.io/download) an
 ```bash
 aws emr create-default-roles
 ```
-2. Upload `integration/emr/alluxio-emr.sh` script to your S3 bucket.
+2. Upload `integration/emr/alluxio-emr.sh` script to your S3 bucket. (This script was not included in versions <= 2.0.
+For these versions, please download the script from
+[our github](https://github.com/Alluxio/alluxio/tree/master/integration/emr/))
```

Comment:
There's a couple caveats to linking directly to master, but this is ok for now.

The EMR script is hard to maintain because the external dependency on the EMR AMI and the software contained within. However, I'm ok with this for now. We should revisit and have a discussion about how to maintain and support EMR.